server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1314,7 +1308,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 			origin,
 			referenceSequenceNumber: message.operation.referenceSequenceNumber,
 			sequenceNumber,
-			term: this.term,
+			term: 1,
```

Comment:
Can you set this to undefined, or not set it instead of making it 1?

Reply:
It has to be set as part of `ISequencedDocumentMessage` under the current definitions.  But later, when the server picks up the protocol-definitions change this will be removed entirely.

We could set to undefined, but since the default checkpoint uses `term: 1` prior to this change, using `1` is closer to current behavior.  I think either could be fine if you have a strong preference, but since it's going to be removed it should be moot soon anyway.